### ISSUE TITLE
Ability to collapse/expand all groups

### ISSUE THREAD

**Issue by zyv** [rdamazio/efis-editor#44]
While testing my checklist imports, I've noticed that this would come in handy when working with large QRH books. I think a minimal implementation might look like this:

* Add button before the new file icon
  * If all expanded, collapse all
  * If all collapsed, expand all
  * If mixed, collapse all

There are probably better ways, just an idea.

**Comment by rdamazio** [comment]
Thanks for the request - this makes sense, I'll add this feature. (no promise on when exactly, but I will)

**Comment by rdamazio** [comment]
I had started on this, but then someone committed a change that modified every file :grin: :grin: so now I have a bunch of merge conflicts :) thankfully jj makes that easier, but still, I won't be able to finish it today.

**Comment by zyv** [comment]
> I had started on this, but then someone committed a change that modified every file 😁 😁 so now I have a bunch of merge conflicts :) thankfully jj makes that easier, but still, I won't be able to finish it today.

Obviously, I've been hit pretty hard by the same problem 😆 🤣 The trick I used was to copy the affected files out of the working copy before pulling in new changes, then copy them back in and run `npm run format:fix`. After that, I only had to roll back a few changes, but that's it, no merge conflict mess! Not sure if you like this better, but this approach has helped me rebase some very messy branches after massive changes.
